cypress/support/hostsfunctions.ts

```diff
@@ -21,12 +21,12 @@ export function createAndCheckHost(host_type: string, inventory: string) {
 
   // create host
   cy.clickButton(/^Create host$/);
-  cy.verifyPageTitle('Create Host');
+  cy.verifyPageTitle('Create host');
   cy.getByDataCy('name').type(hostName);
   cy.getByDataCy('description').type('This is the description');
 
   if (host_type === 'stand_alone_host') {
-    cy.singleSelectByDataCy('inventory', inventory);
+    cy.singleSelectByDataCy('[data-cy="inventory-form-group"]', inventory);
```

Comment:
this one will never match .. `singleSelectByDataCy` already adds the data-cy= part, so...

```suggestion
    cy.singleSelectByDataCy('inventory-form-group', inventory);
```